cloudaux/orchestration/aws/s3.py

```diff
@@ -323,7 +323,7 @@ def get_bucket(bucket_name, output='camelized', include_created=False, **conn):
         'analytics_configurations': get_bucket_analytics_configurations(bucket_name, **conn),
         'metrics_configurations': get_bucket_metrics_configurations(bucket_name, **conn),
         'inventory_configurations': get_bucket_inventory_configurations(bucket_name, **conn),
-        '_version': 4
+        '_version': 5
```

Comment:
Does the version need to be bumped?  We're not changing the format of the response?